R/zzz.R

```diff
@@ -5,3 +5,5 @@
 
 ### global variables
 ggplot_themes <- c("gray", "bw", "linedraw", "light", "dark", "minimal", "classic", "void")
+
+interactive <- NULL
```

Comment:
I can not seem to find where its used?

Reply:
It needs a binding to mock to `base::interactive`, as the `.package` is not being set (mocks multiple functions)

It's being used with `with_mocked_bindings` here:

https://github.com/insightsengineering/teal.modules.general/blob/a5a49d027d032f789a7a6e2f79f0268c9c705cc9/tests/testthat/test-examples.R#L120-L124